src/repositories/invoice.repository.ts

```diff
@@ -0,0 +1,22 @@
+import { OkPacket } from "mysql";
+import { container } from "tsyringe";
+import { Db } from "./Db";
+import { IInvoice } from "../services/invoice.interface";
+export class InvoiceRepository {
+  private db: Db;
+
+  constructor() {
+    this.db = container.resolve<Db>(Db);
+  }
+
+  public async getInvoicesPaginated(pageNumber: number): Promise<IInvoice[]> {
+    const invoicesPerPage: number = 4;
```

Comment:
great !  move this invoicesPerPage to the function parameters.
rename it as itemsPerPage

Reply:
also set a default value to pageNumber = 1

Reply:
done Thanks!